src/components/Settings/SwitchToggle/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
```

Comment:
@naveed-ahmad check https://github.com/quran/common-components/tree/master/src/Toggle

update to `quran-components@0.0.75`